lib/ohai/plugins/linux/network.rb

```diff
@@ -273,6 +273,30 @@ def ethernet_coalesce_parameters(iface)
     iface
   end
 
+  # determine pause parameters for the interface using ethtool
+  def ethernet_pause_parameters(iface)
+    return iface unless ethtool_binary_path
+
+    iface.each_key do |tmp_int|
+      next unless iface[tmp_int][:encapsulation] == "Ethernet"
+
+      so = shell_out("#{ethtool_binary_path} -a #{tmp_int}")
+      Ohai::Log.debug("Plugin Network: Parsing ethtool output: #{so.stdout}")
```

Comment:
```suggestion
      logger.trace("Plugin Network: Parsing ethtool output: #{so.stdout}")
```

This was the only mention of `Ohai::Log` in this file, so it should be changed to match the surrounding functions.

Reply:
I amended my commit with the suggestion. Hope that's fine, I'm not really sure on the GH etiquette :)